src/basic/log.c

```diff
@@ -1291,7 +1291,7 @@ int log_max_levels_to_string(int level, char **ret) {
         for (LogTarget target = 0; target < _LOG_TARGET_SINGLE_MAX; target++) {
```

Comment:
hmm, i don't grok log_max_levels_to_string() on one hand you pass in a level, on the other hand it uses the log_target_max_level[] array? So is it supposed to format the level you specify, or the levels set?